user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleContentComponent.java

```diff
@@ -308,6 +310,16 @@ private void editBatch(EditBatchDialog.ConfirmEvent confirmEvent) {
     });
   }
 
+  private void removeRowFromBatch(EditBatchDialog.RemoveRowEvent removeRowEvent) {
+    EditBatchDialog dialog = removeRowEvent.getSource();
+    if(deletionService.isSampleRemovable(removeRowEvent.getSampleInfo().getSampleId(),
+        context.projectId().orElseThrow())) {
+      dialog.removeRow();
+    } else {
+      dialog.displayRemoveRowError();
```

Comment:
currently this error message will stay forever.

Reply:
How could one handle resetting this in the spreadsheet?

Reply:
done, see above.